### DISCUSSION TITLE
Unable to resolve action. Repository not found: nogsantos/scp-deploy

### DISCUSSION THREAD

**Discussion by ashiquemohammed** [community/community#59086] [Enterprise] [answered]
### Select Topic Area

Question

### Body

I recently moved my github account to enterprise account and since then or not sure if this is the reason , Few of my github actions are failing . Its a little unusual as only some jobs fail . Below to be specific ? 

Unable to resolve action. Repository not found: nogsantos/scp-deploy

**Comment by airtower-luna** [comment, marked as answer]
It seems that the action repository has been renamed, trying to open it redirects here: https://github.com/nogsantos/ssh-scp-deploy

You'll have to update your workflows to use the new name. I recommend you first double-check if that action is what you want, though.

**Comment by ashiquemohammed** [comment]
@airtower-luna  

Thank you for your answer . It was right , I was shocked a well maintained repo forgot to update their docs . 

@https://github.com/nogsantos/

https://github.com/nogsantos/
